app/scripts/components/form/text-area.js

```diff
@@ -33,6 +33,12 @@ const TextAreaForm = React.createClass({
     this.props.onChange(this.props.id, value);
   },
 
+  setEditorRef: function (editorRef) {
+    if (window.Cypress) {
+      window.aceEditorRef = editorRef;
```

Comment:
This is what makes it all work: we're exposing the editor from the actual editor component, which allows us to use it to `getValue` and `setValue` directly instead of using unreliable approaches with `cy.type`.